Cql/Cql.Firely/FhirTypeConverter.cs

```diff
@@ -18,8 +18,10 @@
 
 namespace Hl7.Cql.Fhir
 {
-    internal static class FhirTypeConverter
+    public static class FhirTypeConverter
```

Comment:
Can you remove the 

`#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member` 

from the top of the file and document this class?

Reply:
Sure,  I will update it in a subsequent PR.